slscore/SLSManager.cpp

```diff
@@ -284,17 +284,24 @@ int  CSLSManager::check_invalid()
     return SLS_ERROR;
 }
 
-void CSLSManager::get_stat_info(std::string &info)
+void CSLSManager::get_stat_info(std::string &info_str)
 {
+    info_str += '[';
```

Comment:
Great job debugging and fixing this!

JSON response best practice is to always return a JSON object, not just an Array. I recommend that when sending this data in the stats request body, it be wrapped in an object like `{ workers: [...workers] }`. This way, you can extend the data returned later without making breaking changes.

For reference, this is what I'm seeing in the request body:
```
[
  {
    port: '8080',
    role: 'player',
    pub_domain_app: 'input/live',
    stream_name: 'roger',
    url: 'output/live/roger',
    remote_ip: '192.168.1.114',
    remote_port: '61426',
    start_time: '2020-11-03 11:35:37',
    kbitrate: '11033'
  },
  {
    port: '8080',
    role: 'publisher',
    pub_domain_app: 'input/live',
    stream_name: 'roger',
    url: 'input/live/roger',
    remote_ip: '192.168.1.114',
    remote_port: '64291',
    start_time: '2020-11-03 11:35:27',
    kbitrate: '11765'
  },
  {
    port: '8080',
    role: 'listener',
    pub_domain_app: '',
    stream_name: '',
    url: '',
    remote_ip: '',
    remote_port: '',
    start_time: '2020-11-03 11:35:08',
    kbitrate: '0'
  }
]
```

Reply:
Also, at least for me the response now starts with "[,{ port: ..." in which the comma of course breaks any JSON parsing. Great direction though!

Reply:
I don't expect any pull requests to be merged. It would be great if someone interested in maintaining the project would fork it, because there are a lot of very small improvements that would make it much more flexible and usable in different scenarios. I've personally moved on to a different solution for SRT streaming, but this project was a great help to me initially.

Reply:
Agreed, this project was one of the first I noticed and really got me going for open source SRT streaming.  People get busy and such, so it's understandable that not everyone has time to keep up with their stuff.  Luckily, it's inspired others to do their own versions - if it's ok to mention other projects here, I've been following voc/srtrelay (which is based in part on sls). It's in golang so a little more merciful to my wheelhouse.